zammad/Chart.yaml

```
@@ -1,10 +1,10 @@
 apiVersion: v2
 name: zammad
-version: 6.0.4
-appVersion: 5.0.3
+version: 6.0.5
```

i would go up one minor version here too as this is no bugfix.
```suggestion
version: 6.1.0
```